version.php

```diff
@@ -18,15 +18,15 @@
  * The version file for qtype_algebra
  *
  * @package    qtype_algebra
- * @copyright  Roger Moore <rwmoore@ualberta.ca>
```

Comment:
seems a bit harsh to remove them, can't we list multiple contributors?

Reply:
have amended now.